src/Microsoft.OData.Core/Uri/ODataUri.cs

```diff
@@ -30,13 +30,13 @@ public sealed class ODataUri
         /// </summary>
         private Uri serviceRoot;
 
+        private ParameterAliasValueAccessor parameterAliasValueAccessor;

```

Comment:
Line 130, ParameterAliasNodes has a null check on the property. That null check does not seem required? Or if it is, it should be replaced with the field, instead of the property?

Reply:
This is for the ParameterAliasNodes getter - it has a null check for ParameterAliasValueAccessor which is redundant in the current iteration, since ParameterAliasValueAccessor is never null. Hence should check against the **field** instead of the **property**, or perhaps remove the null check.